### DISCUSSION TITLE
[Esame 9 settembre 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#612] [Esami M-Z]
![4](https://github.com/user-attachments/assets/f6623151-df6d-472b-9907-9a1c07c3295b)

**Comment by AglaiaNorza** [comment]
a me viene così
![image](https://github.com/user-attachments/assets/769d4e38-5382-440b-9e6f-7297351b9bf8)

**Reply by Sampyx** [reply]
X3
